doc/user_guide.md

```diff
@@ -0,0 +1,324 @@
+
+<img src="file:///C:/adftut/doc/img/2021-08-04-15-28-19-image.png" title="" alt="" width="747">
+
+### Alternate approach : Deploying to Azure using Visual Studio or Visual Code ''Publish''
+
+These methods may be a bit more involved and require you to have visual studio or visual studio code installed but might be more practical if you want to alter or add some of the behaviour of these functions yourself.
+
+You'll find guides for visual studio here: https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-your-first-function-visual-studio and for visual studio code here: [Create a C# function using Visual Studio Code - Azure Functions | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-csharp) 
+
+
+
+## Using the functions in Azure Data factory.
+
+If you haven't done so yet, create a ''Data Factory' in azure.
+
+Open ''Azure Data Factory Studio'.
+
+### Setting up a linked service to your function app
+
+Let's set up a linked service to our function app so we can use these functions in our pipelines.
+
+Click on the ''manage' 'tab icon (wrench in a toolbox icon) to the left.
+
+Under 'Connections' Select 'Linked services', then Click ''New'.
+
+![](./img/2021-08-04-16-19-59-image.png)
+
+Click the Compute tab at the top, select 'Azure Function'.
+
+![](./img/2021-08-04-16-20-15-image.png)
+
+We have some configuration to do:
+
+Pick a  good name,
+
+as integration runtime, select the 'AutoResolveIntegrationRuntime',
+
+select your azure subscription and your azure function app url (both are dropdowns).
+
+The last thing you got to fill in is the 'Function key'.
+
+ This key is used to prevent others from accessing your azure functions and to authenticate your requests from within azure data factory.
+
+You can find this key (or create an additional one) in your Function App under Settings:
+
+![](./img/2021-08-04-16-25-39-image.png)
+
+Open a new tab in your browser and navigate to your Function App you set up in the earlier steps.
+
+In your function app, select the 'App keys' tab. 
+
+Under 'Host keys' you'll see the default key. We'll use this key, if you wish you could also create an additional key and use that one.
+
+Click the eye icon and then the copy icon to copy the key to your clipboard. Paste this key into the Function key field of the new linked service we're creating.
+
+![](./img/2021-08-04-16-21-50-image.png)
+
+Click ''Create' to create the linked service.
+
+Click 'Publish all' at the top
+
+This concludes the setup phase. We're now ready to start using these functions in our pipelines.
+
+Open the 'Author' tab.
+
+### Using the exasol azure functions in your pipelines
+
+All 3 functions are currently implemented using POST methods where you send a JSON body containing the parameters.
+
+#### Query method
+
+Let's start with using the Query method:
```

Comment:
I would avoid colon here (and in other places). A lowercase letter is usually used after colon, so it should be the same sentence